eppo_core/src/eval/eval_precomputed_assignments.rs

```diff
@@ -2,29 +2,58 @@ use std::collections::HashMap;
 use std::sync::Arc;
 
 use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
 
 use crate::eval::get_assignment;
-use crate::ufc::Assignment;
-use crate::ufc::{AssignmentFormat, Environment};
-use crate::{Attributes, Configuration, EvaluationError, Str};
+use crate::ufc::{Assignment, AssignmentFormat, Environment, ValueWire, VariationType};
+use crate::{Attributes, Configuration, Str};
 
-#[derive(Debug)]
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct PrecomputedConfiguration {
-    pub created_at: DateTime<Utc>,
-    pub format: AssignmentFormat,
-    pub environment: Environment,
-    pub flags: HashMap<String, Result<Assignment, EvaluationError>>,
+    created_at: DateTime<Utc>,
```

Comment:
Another change here is that all fields are private to `eppo_core`: edge server shouldn't need to tinker with them and clients would have their own parsers.

Basically, the only way to interact with `PrecomputedConfiguration` is via serialize/deserialize.

This should allow us tweaking the type here without being afraid breaking the edge server.